lib/orchestrator/application.ex

```diff
@@ -104,30 +101,13 @@ defmodule Orchestrator.Application do
 if Mix.env() == :test do
   # For now, the simplest way to make tests just do tests, not configure/start anything.
   defp filter_children(_children), do: []
-  defp configure_api_token, do: :ok
-  defp configure_slack_reporter, do: :ok
 else
-  defp filter_children(children), do: children
-  defp configure_api_token do
-    token =
-      case System.get_env("CANARY_API_TOKEN") do
-        nil ->
-          "fake-token-for-dev"
-
-        token ->
-          Orchestrator.Configuration.translate_value(token)
-      end
-
-    Application.put_env(:orchestrator, :api_token, token)
-  end
-  defp configure_slack_reporter do
-    token = System.get_env("SLACK_API_TOKEN")
-    |> Orchestrator.Configuration.translate_value()
-    Application.put_env(:slack, :api_token, token)
```

Comment:
There is no `:slack` application (and Mix config complains if you add a `config :slack` app) so I moved this to `slack_api_token` under the orchestrator configs.